### PR TITLE
Add content to install application page

### PR DESCRIPTION
**Pull request by pipo02mix**
### What this PR does / why we need it

Towards https://github.com/giantswarm/giantswarm/issues/30831

### Things to check/remember before submitting

- If you made content changes

    - Run `make lint dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/getting-started/install-an-application/_index.md [hunk 1]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+test01-security-bundle                 1.8.0               12m          3m32s           deployed
+test01-teleport-kube-agent             0.9.2               12m          4m24s           deployed
+test01-vertical-pod-autoscaler         5.2.4               12m          110s            deployed
+```
+
+__Note__: We don't enforce the cluster prefix (here: `test01-`), but it's a good practice to have it.
+
+You can see that several applications already exist for the workload cluster `test01`. Most of the apps run directly in the workload cluster itself. The operator which actually deploys the applications to the workload clusters is running on the management cluster and is called `app-operator`. Learn more about this process [in this guide]({{< relref "/vintage/platform-overview/app-platform/" >}}).
+
+## Step 2: Install an ingress nginx controller
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:66]:
```suggestion
## Step 2: Install an ingress nginx controller {#install-ingress-controller}
```

File: src/content/getting-started/install-an-application/_index.md [hunk 2]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+Now you can access your workload cluster and check the app running and see which ingress URL has been generated:
+
+```nohighlight
+kubectl get ingress -n default
+NAME          CLASS   HOSTS                                        ADDRESS                                                                       PORTS     AGE
+hello-world   nginx   cluster.provider.k8s.giantswam.io   ab49484.cn-north-1.elb.amazonaws.com.cn   80, 443   2m29s
+```
+
+Sadly the URL isn't accessible because we've not passed the right domain to the ingress controller.
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:184]:
```suggestion
Sadly, the address isn't accessible because we've not passed the right domain on to the ingress controller.
```

File: src/content/getting-started/install-an-application/_index.md [hunk 3]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+After few seconds you will see the deployed status:
+
+```nohighlight
+kubectl get app test01-hello-world
+NAME                INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world   2.3.2               11s          8s              deployed
+```
+
+Now you can access your workload cluster and check the app running and see which ingress URL has been generated:
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:176]:
```suggestion
Now you can access your workload cluster, check the app running, and see which ingress address has been generated:
```

**Comment by pipo02mix** [comment]
@AndiDog do you have time today?

File: src/content/getting-started/install-an-application/_index.md [hunk 4]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+Sadly, the address isn't accessible because we've not passed the right domain on to the ingress controller.
+
+## Step 4: Configuring the right domain
+
+For every workload cluster there is a default wildcard record created automatically by our controllers. The domain is composed by the cluster name and the provider, the region and the base domain.
+
+```nohighlight
+kubectl get cm test01-cluster-values -ojsonpath="{.data.values}" | yq .baseDomain
+test01.capi.aws.k8s.gigantic.io
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:192]:
````suggestion
```sh
$ kubectl get cm -n org-testing test01-cluster-values -ojsonpath="{.data.values}" | grep baseDomain
baseDomain: test01.capi.aws.k8s.gigantic.io
````

File: src/content/getting-started/install-an-application/_index.md [hunk 5]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+```
+
+With that information we can customize our `hello-world` app to use the right domain. Checking the [app values schema](https://github.com/giantswarm/hello-world-app/blob/main/helm/hello-world/values.schema.json) we can see the `host` property should override in `hosts` and `tls` sections.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:195]:
```suggestion
With that information, you can customize the `hello-world` app to use the right domain. The [app values schema](https://github.com/giantswarm/hello-world-app/blob/main/helm/hello-world/README.md) documents which values can be changed. Here, the `hosts` and `tls` sections are relevant.
```

File: src/content/getting-started/install-an-application/_index.md [hunk 6]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+Lets create a file with the values we want to override:
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:197]:
```suggestion
Create a file containing the Helm chart values:
```

File: src/content/getting-started/install-an-application/_index.md [hunk 7]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+```nohighlight
+kubectl get app test01-hello-world
+NAME                                  INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world                     2.3.2               51m          8s             deployed
+```
+
+At this point you should be able to access the `hello-world` app using the domain you have configured. Check the `URL` value in the ingress resource has been updated to be sure the changes have been propagated. Now you should be able to see the hello world frontend.
+
+![Hello world page](hello-world.png)
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:237]: This should at best be consistent with our cluster naming (`test01`), even if you only just edit the address bar to fake it

File: src/content/getting-started/install-an-application/_index.md [hunk 8]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+- host: hello.test01.capi.pek.aws.k8s.adidas.com.cn
+  paths:
+  - path: /
+    pathType: Prefix
+tls:
+- secretName: hello-world-tls
+  hosts:
+  - hello.test01.capi.pek.aws.k8s.adidas.com.cn
+EOL
+```
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:212]:
````suggestion
```sh
cat > hello-world-values.yaml <<EOF
ingress:
hosts:
- host: hello.test01.capi.aws.k8s.gigantic.io
  paths:
  - path: /
    pathType: Prefix
tls:
- secretName: hello-world-tls
  hosts:
  - hello.test01.capi.aws.k8s.gigantic.io
EOF
```
````

File: src/content/getting-started/install-an-application/_index.md [hunk 9]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+Now we can template the app again passing the file reference:
+
+```nohighlight
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:216]:
````suggestion
```sh
````

File: src/content/getting-started/install-an-application/_index.md [hunk 10]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+kubectl get ingress -n default
+NAME          CLASS   HOSTS                                        ADDRESS                                                                       PORTS     AGE
+hello-world   nginx   cluster.provider.k8s.giantswam.io   ab49484.cn-north-1.elb.amazonaws.com.cn   80, 443   2m29s
+```
+
+Sadly, the address isn't accessible because we've not passed the right domain on to the ingress controller.
+
+## Step 4: Configuring the right domain
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:186]: These sections must be merged, and the "oh sorry we described an incomplete step" part must go away. Tutorials must be concise and correct, explaining traps before they happen.

Let's remove the above app templating (including the China domain) and go on below.

File: src/content/getting-started/install-an-application/_index.md [hunk 11]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+For every workload cluster there is a default wildcard record created automatically by our controllers. The domain is composed by the cluster name and the provider, the region and the base domain.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:188]: This isn't comprehensible with such a standalone description

File: src/content/getting-started/install-an-application/_index.md [hunk 12]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+  --catalog=giantswarm \
+  --cluster-name=test01 \
+  --organization=giantswarm \
+  --name=test01-hello-world \
+  --target-namespace=default \
+  --user-configmap=hello-world-values.yaml \
+  --version=2.3.2 > hello-world.yaml
+```
+
+We push the file or apply the changes to the platform API and wait a couple of seconds to see the changes:
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:227]:
````suggestion
Again, push the generated `App` CR file to your GitOps repository, or apply it directly to the management cluster with the following command:

```sh
kubectl apply -f hello-world.yaml
```
````

File: src/content/getting-started/install-an-application/_index.md [hunk 13]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+```nohighlight
+kubectl get app test01-hello-world
+NAME                                  INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world                     2.3.2               51m          8s             deployed
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:232]:
```suggestion
$ kubectl get app test01-hello-world
NAME                INSTALLED VERSION  CREATED AT  LAST DEPLOYED  STATUS
test01-hello-world  2.3.2              51m         8s             deployed
```

File: src/content/getting-started/install-an-application/_index.md [hunk 14]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+```
+
+At this point you should be able to access the `hello-world` app using the domain you have configured. Check the `URL` value in the ingress resource has been updated to be sure the changes have been propagated. Now you should be able to see the hello world frontend.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:235]: If both the ingress-nginx controller and the hello-world app are healthy in the workload cluster, you should be able to access the hello world web page at `https://test01.capi.aws.k8s.gigantic.io` (please fill in your own base domain).

Based on the above `Ingress` definition in the Helm values, a TLS certificate should have been automatically generated using cert-manager, which is installed by default. Since the trusted certificate authority Let's Encrypt is used by default, your browser should show the web page as secure, without warnings.

File: src/content/getting-started/install-an-application/_index.md [hunk 15]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+![Hello world page](hello-world.png)
+
+__Note__:You can read more about app platform configuration [here]({{< relref "/vintage/getting-started/app-platform/app-configuration" >}}).
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:239]: Should we put this at the end (`## Further reading` or so)?

File: src/content/getting-started/install-an-application/_index.md [hunk 16]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+## Step 5: Deleting the hello-world app
+
+The deletion of an app is as simple as creating it. You can delete the app by running:
+
+```nohighlight
+kubectl delete -f hello-world.yaml
+kubectl delete -f ingress-nginx.yaml
+```
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:248]:
````suggestion
## Step 5: Deleting the hello-world app and ingress-nginx controller

The deletion of an app is as simple as creating it:

```nohighlight
kubectl delete -f hello-world.yaml

kubectl delete -f ingress-nginx.yaml
```
````

File: src/content/getting-started/install-an-application/_index.md [hunk 17]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+## Next step
+
+After learn how to deploy an application in the workload cluster, lets bump into [a different connectivity options you can rely on within the platform]({{< relref "/getting-started/expose-your-app" >}}).
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:252]: Grammar – can you please reword?

And can we include a lookout what the target article is actually about? The description "different connectivity options" is very vague.

File: src/content/getting-started/install-an-application/_index.md [hunk 18]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+## Step 2: Install an ingress nginx controller
+
+Before we install our publicly accessible `hello-world` app, we need to have an ingress controller running in the cluster. The ingress controller is responsible for routing the incoming traffic to the correct service in the cluster and make it available to the public.
+
+To know which applications are available for customers we've extended the platform with two custom resources. First resource is the [`AppCatalog`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogs.application.giantswarm.io/" >}}) which is a recipient to collect application definitions that are available to install in the workload clusters. The second is the [`AppCatalogEntry`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) which is the representation of the application definition which has a version defined.
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:70]: But we dont want to promote happa anymore, it is deprecated and I did want to not mention anymore

File: src/content/getting-started/install-an-application/_index.md (hunk 15 above)
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:243]: I dont like it. I prefer to link to tutorials or more extensive documents across the content instead of doing it further down. At least in getting started. THe main goal in this section is to go from nothing to a full, complete dev platform journey for beginners

File: src/content/getting-started/install-an-application/_index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:70]: @AndiDog WDYT?

File: src/content/getting-started/install-an-application/_index.md [hunk 19]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+````
+
+In our case we're interested in the ingress nginx controller, so let's check the latest version available in the catalog:
+
+```nohighlight
+kubectl get appcatalogentries -n default -l app.kubernetes.io/name=ingress-nginx
+NAME                             CATALOG      APP NAME        VERSION   UPSTREAM VERSION   AGE
+giantswarm-ingress-nginx-3.9.2   giantswarm   ingress-nginx   3.9.2     1.11.2             2d4h
+...
+```
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:95]: Well, since this is an introduction, it is useful to introduce the concept of catalogue and entry. And I favour using a single command with a flag to filter then chaining tools. That's probably something we need to agree

File: src/content/getting-started/install-an-application/_index.md [hunk 20]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+
+The next step is deploying the `hello-world` application using the same approach as we did with the ingress controller.
+
+First we list the versions with:
+
+```nohighlight
+kubectl get appcatalogentries -n default -l app.kubernetes.io/name=hello-world
+NAME                           CATALOG      APP NAME      VERSION   UPSTREAM VERSION   AGE
+...
+giantswarm-hello-world-2.3.2   giantswarm   hello-world   2.3.2     0.2.1              72d
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:134]: same here

File: src/content/getting-started/install-an-application/_index.md [hunk 21]
```diff
@@ -15,4 +15,238 @@ user_questions:
...
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: hello-world
+  namespace: kube-system
+  version: 2.3.2
+```
+
+The `name` field is the name of the app in the catalog meanwhile `--app-name` designates the name of the app instance installed in the cluster. Keep in mind that the app name is subject to different length limits, depending on how the app is deployed. Using a name under 30 characters is recommended.
+
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:167]: ok let me rephrase it

File: src/content/getting-started/install-an-application/_index.md (hunk 10 above)
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:186]: ok I will do that

File: src/content/getting-started/install-an-application/_index.md [hunk 22]
```diff
@@ -15,4 +15,245 @@ user_questions:
...
+```nohighlight
+kubectl delete -f hello-world.yaml
+
+kubectl delete -f ingress-nginx.yaml
+```
+
+## Next step
+
+After learn how to deploy an application in the workload cluster, lets bump into [a different connectivity options you can rely on within the platform]({{< relref "/getting-started/expose-your-app" >}}).
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:259]:
```suggestion
After learning how to deploy an application in the workload cluster, let's dive into [in how networking works in your cluster and how to configure to harden your application]({{< relref "/getting-started/expose-your-app" >}}).
```

File: src/content/getting-started/install-an-application/_index.md [hunk 23]
```diff
@@ -15,4 +15,245 @@ user_questions:
...
+spec:
+  catalog: giantswarm
+  kubeConfig:
+    inCluster: false
+  name: hello-world
+  namespace: kube-system
+  version: 2.3.2
+```
+
+The `name` field is the name of the app in the catalog meanwhile `--app-name` designates the name of the app instance installed in the cluster. Keep in mind that the app name is subject to different length limits, depending on how the app is deployed. Using a name under 30 characters is recommended.
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:166]:
```suggestion
The `spec.name` field is the application's name in the catalog; meanwhile, 'metadata.name` designates the name of the instance installed in the cluster. [Read more information about the properties here]({{ relfer "/vintage/use-the-api/management-api/crd/apps.application.giantswarm.io/" }}).
```

File: src/content/getting-started/install-an-application/_index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (AndiDog) [reply src/content/getting-started/install-an-application/_index.md:70]: Ok sure, then let's leave out happa and only have the platform API example remaining

File: src/content/getting-started/install-an-application/_index.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (AndiDog) [reply src/content/getting-started/install-an-application/_index.md:95]: Sure, what I wrote are only suggestions. Please resolve any of my comments which you think are done/unneeded. Then I can go on reviewing and approving this.

File: src/content/getting-started/provision-your-first-workload-cluster/_index.md [hunk 24]
```diff
@@ -50,6 +51,7 @@ kubectl gs template cluster \
   --provider capa \
   --name mycluster \
   --organization testing \
+  --release 29.0.0 \
```
Comment (pipo02mix) [inline src/content/getting-started/provision-your-first-workload-cluster/_index.md:54]:
```suggestion
  --release 29.1.0 \
```

File: src/content/getting-started/install-an-application/_index.md [hunk 25]
```diff
@@ -15,4 +15,252 @@ user_questions:
   - What do I need to do to configure the app platform to build my capabilities?
 ---
 
-<!-- Choice of apps from catalog (e.g. via happa), configuration with values (often defaults are recommended), installation with kubectl-gs templating and apply (we should use a very commonly used example such as nginx-ingress or ESO) -->
+The _Giant Swarm App Platform_ is built on top of [Helm](https://helm.sh/) charts and allows you to manage apps and their configurations represented by `App` custom resources (CRs) for multiple clusters, from a single place: the [Platform API]({{< relref "/overview/architecture#platform-api" >}}) on the management cluster.
+
+In this guide, we will install a `hello-world` app together with an Ingress NGINX Controller to serve the web application publicly. We will do this by using kubectl, to create an [App]({{< relref "/vintage/use-the-api/management-api/crd/apps.application.giantswarm.io.md" >}}) CR using the platform API of your management cluster.
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:20]:
```suggestion
In this guide, we will install a `hello-world` app together with an ingress nginx controller to serve the web application publicly. We will do this by using kubectl, to create an [App]({{< relref "/vintage/use-the-api/management-api/crd/apps.application.giantswarm.io.md" >}}) CR using the platform API of your management cluster.
```

File: src/content/getting-started/install-an-application/_index.md [hunk 26]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+## Step 2: Install an ingress nginx controller {#install-ingress-controller}
+
+Before installing the `hello-world` app, an ingress controller must be running in the cluster. The ingress controller is responsible for routing the incoming traffic to the correct service in the cluster and make it available publicly.
+
+To know which applications are available for customers we've extended the platform with two custom resources. First resource is the [`AppCatalog`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogs.application.giantswarm.io/" >}}) which is a recipient to collect application definitions that are available to install in the workload clusters. The second is the [`AppCatalogEntry`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) which is the representation of the application definition which has a version defined.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:70]:
```suggestion
To know which applications are available for customers we've extended the platform with two custom resources. The first resource is [`AppCatalog`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogs.application.giantswarm.io/" >}}) which contains application definitions that are available for installation in the workload clusters. The second is the [`AppCatalogEntry`]({{< relref "/vintage/use-the-api/management-api/crd/appcatalogentries.application.giantswarm.io.md" >}}) which is the application definition for each application version.
```

File: src/content/getting-started/install-an-application/_index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (AndiDog) [reply src/content/getting-started/install-an-application/_index.md:70]: The above suggestion still holds important diffs, such as `AppCatalog` (old name) vs. `Catalog`, having `$ command` in the first line of the snippets, etc. Please take another look what should be taken over, even if we decided against mentioning happa.

Same below where text snippets should have `$ ` before the command – if and only if there's both command _and_ output in the same snippet. That is inconsistent also in the other sections of the article. We should also use the typical `[...]` continuation format rather than `...`, so it's clear where we cut out the remaining output.

Please try out all snippets since I'm not familiar with querying these things on the command line, and can't easily review which are deprecated CRs and which are new ones.

File: src/content/getting-started/install-an-application/_index.md [hunk 27]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+```nohighlight
+kubectl get ingress -n default
+NAME          CLASS   HOSTS      ADDRESS      PORTS     AGE
+hello-world   nginx   hello.test01.capi.aws.k8s.gigantic.io   ab49484...   80, 443   2m29s
+```
+
+It's time to try to reach the application using the ingress:
+
+```sh
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:246]:
```suggestion
```nohighlight
```

File: src/content/getting-started/install-an-application/_index.md [hunk 28]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+/ Accept-Ranges: bytes ...
+```
+
+The `curl` request it's routed through the AWS load balancer to the ingress controller, which then forwards the request to the `hello-world` service.
+
+## Step 4: Deleting the hello-world app and ingress-nginx controller
+
+The deletion of an app is as simple as creating it:
+
+```nohighlight
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:258]:
````suggestion
```sh
````

File: src/content/getting-started/install-an-application/_index.md [hunk 29]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+First, list the versions with:
+
+```nohighlight
+kubectl get appcatalogentries -n default -l app.kubernetes.io/name=hello-world
+NAME                           CATALOG      APP NAME      VERSION   UPSTREAM VERSION   AGE
+...
+giantswarm-hello-world-2.3.2   giantswarm   hello-world   2.3.2     0.2.1              72d
+```
+
+To make our app accessible from Internet an [ingress]() should be defined with the right domain. By default every workload cluster comes with a default wildcard record created automatically by our controllers. The domain is composed by the cluster name and the provider, the region and the base domain.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:137]:
```suggestion
To make our app accessible from the internet, an [ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) should be defined with the right domain. By default every workload cluster comes with a default wildcard DNS record (e.g. `*.test01.capi.aws.k8s.gigantic.io`). The domain is composed by the cluster name and the management cluster's base domain.

You can find the domain of your workload cluster like this:
```

(removing wrong information; I guess this was copied from vintage)

File: src/content/getting-started/install-an-application/_index.md [hunk 30]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+```
+
+Lets take a look at the manifests generated to learn how they look like:
+
+```yaml
+apiVersion: v1
+data:
+  values: |
+    ingress:
+      ...
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:183]:
```suggestion
      [...]
```

(repeated)

File: src/content/getting-started/install-an-application/_index.md [hunk 31]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+  name: hello-world
+  namespace: kube-system
+  userConfig:
+    configMap:
+      name: test01-hello-world-userconfig
+      namespace: org-testing
+  version: 2.3.2
+```
+
+The `spec.name` field is the application's name in the catalog; meanwhile, 'metadata.name` designates the name of the instance installed in the cluster. [Read more information about the properties here]({{ relref "/vintage/use-the-api/management-api/crd/apps.application.giantswarm.io/" }}).
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:206]:
```suggestion
The `spec.name` field is the application's name in the catalog, while `metadata.name` designates the name of the `App` object in the cluster and can therefore be freely chosen. [Read more information about the properties here]({{ relref "/vintage/use-the-api/management-api/crd/apps.application.giantswarm.io/" }}).
```

File: src/content/getting-started/install-an-application/_index.md [hunk 32]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+Now let's push the file to your GitOps repository, or apply it directly to the platform API with the following command:
+
+```sh
+kubectl apply -f hello-world.yaml
+```
+
+```nohighlight
+$ kubectl get app test01-hello-world
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:215]:
```suggestion
$ kubectl get app -n org-testing test01-hello-world
```

(please check if the missing `-n` parameter is repeated)

File: src/content/getting-started/install-an-application/_index.md [hunk 33]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+NAME                INSTALLED VERSION  CREATED AT  LAST DEPLOYED  STATUS
+test01-hello-world  2.3.2              51m         8s             deployed
+```
+
+If both the ingress-nginx controller and the hello-world app are healthy in the workload cluster, you should be able to access the hello world web page at `https://test01.capi.aws.k8s.gigantic.io` (please fill in your own base domain).
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:220]:
```suggestion
If both the ingress-nginx controller and the hello-world app are healthy in the workload cluster, you should be able to access the hello world web page at `https://hello.test01.capi.aws.k8s.gigantic.io` (please fill in your own workload cluster domain).
```

File: src/content/getting-started/install-an-application/_index.md [hunk 34]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+After a few seconds, you should see the deployed status:
+
+```nohighlight
+$ kubectl get app -n org-testing test01-hello-world
+NAME                INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world   2.3.2               11s          8s              deployed
+```
+
+Now you can access your workload cluster, check the app running, and see which ingress address has been generated:
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:236]: What's an "ingress address"? Rather

```
`Ingress` object
```

?

File: src/content/getting-started/install-an-application/_index.md [hunk 35]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+It's time to try to reach the application using the ingress:
+
+```sh
+$ curl -Is https://hello.test01.capi.aws.k8s.gigantic.io
+/ HTTP/1.1 200 OK
+/ Accept-Ranges: bytes ...
+```
+
+The `curl` request it's routed through the AWS load balancer to the ingress controller, which then forwards the request to the `hello-world` service.
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:252]:
```suggestion
The `curl` request is routed through the AWS load balancer to the ingress controller, which then forwards the request to the `hello-world` service.
```

File: src/content/getting-started/install-an-application/_index.md [hunk 36]
```diff
@@ -15,4 +15,252 @@ user_questions:
...
+
+```nohighlight
+kubectl delete -f hello-world.yaml
+
+kubectl delete -f ingress-nginx.yaml
+```
+
+## Next step
+
+After learning how to deploy an application in the workload cluster, let's dive into [in how networking works in your cluster and how to configure to harden your application]({{< relref "/getting-started/expose-your-app" >}}).
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:266]:
```suggestion
After learning how to deploy an application in the workload cluster, let's dive into [in how networking works in your cluster and how to configure security hardening of your application]({{< relref "/getting-started/expose-your-app" >}}).
```

File: src/content/getting-started/install-an-application/_index.md [hunk 37]
```diff
@@ -15,4 +15,254 @@ user_questions:
...
+
+After a few seconds, you should see the deployed status:
+
+```nohighlight
+$ kubectl get app -n org-testing test01-hello-world
+NAME                INSTALLED VERSION   CREATED AT   LAST DEPLOYED   STATUS
+test01-hello-world   2.3.2               11s          8s              deployed
+```
+
+Now you can access your workload cluster, check the app running, and see which ingress address has been generated:
```
Comment (pipo02mix) [inline src/content/getting-started/install-an-application/_index.md:238]:
```suggestion
Now you can access your workload cluster, check the app running, and see which `Ingress` resource has been generated:
```

File: src/content/getting-started/install-an-application/_index.md (hunk 18 above)
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md]: ok I have tried my best, and I also fixed other getting started pages to be consistent

For happa, we leave it out in new docs. We want to add Backstage as soon as it is available to all our customers (current quarter)

File: src/content/getting-started/install-an-application/_index.md [hunk 38]
```diff
@@ -81,22 +75,22 @@ __Note__: You can [create your own catalog]({{< relref "/vintage/getting-started
 
 To browse which applications are available in the catalog you can run the following command:
 
-```nohighlight
+```text
```
Comment (AndiDog) [inline src/content/getting-started/install-an-application/_index.md:78]: It's the other way around – pure commands should be highlighted with `sh`, command line + comand output should be `text` (or `nohighlight` which should be the same).

Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md:78]: ups sorry I mixed them up, yeah is what I wanted

Comment (pipo02mix) [reply src/content/getting-started/install-an-application/_index.md]: fixed

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 39]
```diff
@@ -70,15 +70,15 @@ Alternatively, you can create your role assignment with restricted access to spe
...
-                         --parameters delegatedResourceManagement.parameters.json \
-                         --verbose
-    ```
+```sh
+$ az deployment create --name "giantswarm-access" \
+                      --location <AzureRegion> \
+                      --template-file delegatedResourceManagement.json \
+                      --parameters delegatedResourceManagement.parameters.json \
+                      --verbose
+```
```
Comment (AndiDog) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:79]: This was meant to be indented as part of the bullet point list

````suggestion
    ```sh
    $ az deployment create --name "giantswarm-access" \
                          --location <AzureRegion> \
                          --template-file delegatedResourceManagement.json \
                          --parameters delegatedResourceManagement.parameters.json \
                          --verbose
    ```
````

File: src/content/getting-started/prepare-your-provider-account/azure/_index.md [hunk 40]
```diff
@@ -70,15 +70,15 @@ Alternatively, you can create your role assignment with restricted access to spe
...
 
-    The `--name` value can be freely chosen if you prefer another name.
+The `--name` value can be freely chosen if you prefer another name.
```
Comment (AndiDog) [inline src/content/getting-started/prepare-your-provider-account/azure/_index.md:81]:
```suggestion
    The `--name` value can be freely chosen if you prefer another name.
```

**Review by AndiDog — APPROVED** [review]
